content/postgresql/concepts/constraints/constraints.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: 'constraints'
+Description: 'Constraints are defined for tables or columns in PostgreSQL. Constraints can be used to control data in order to define limits for inserting, updating and deleting data.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Constraints'
+  - 'Database'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'getting-started-off-platform-for-data-science'
+  - 'paths/design-databases-with-postgresql'
+  - 'paths/computer-science'
```

Comment:
```suggestion
CatalogContent:
  - 'getting-started-off-platform-for-data-science'
  - 'paths/design-databases-with-postgresql'
  - 'paths/computer-science'
```
The paths which are present here in the CatalogContent section must be taken  from the Catalog-content.md file